Jenkinsfile.groovy

```diff
@@ -73,7 +77,8 @@ node('docker&&multicore&&ram') {
       try {
         echo("Start all the containers (including tests)")
         sh "docker-compose -v"
-          composeCmd(composeProject, "up -d db minio redis backend db_init worker nginx tests")
+        composeCmd(composeProject, "up -d db minio redis backend db_init worker nginx")
+        sleep 20
```

Comment:
what are those sleep commands for? And is 20 the right value?

Reply:
I lowered and moved the 30 second sleep from line 90 (before following the output of the integration test container). I suspect it gives a bit of buffer for pulling docker images.